pkgs/snaplet/cli/registry.yaml

```diff
@@ -0,0 +1,24 @@
+packages:
+  - name: snaplet/cli
+    type: http
+    description: Seed your PostgreSQL database with production-like data so you can code, debug, and test with ease.
```

Comment:
```suggestion
    description: Seed your PostgreSQL database with production-like data so you can code, debug, and test with ease
```

nits. Please remove a period `.` from the end of the description for consistency.

https://aquaproj.github.io/docs/develop-registry/registry-style-guide/#remove-characters--from-the-end-of-the-description